vlib/rand/README.md

```diff
@@ -0,0 +1,46 @@
+## Quickstart
+
+The V `rand` module provides two main ways in which users can generate pseudorandom numbers:
+
+1. Through top-level functions in the `rand` module.
+   - `import rand` - Import the `rand` module.
+   - `rand.seed(seed_data)` to seed (optional).
+   - Use `rand.int()`, `rand.u32n(max)`, etc.
+2. Through a generator of choice. The PRNGs are included in their respective submodules.
+   - `import rand.pcg32` - Import the module of the PRNG required.
+   - `mut rng := pcg32.PCG32RNG{}` - Initialize the struct. Note that the **`mut`** is important.
+   - `rng.seed(seed_data)` - optionally seed it with an array of `u32` values.
+   - Use `rng.int()`, `rng.u32n(max)`, etc.
+
+## General Background
```

Comment:
User `#` instead of `##`. It's the top level title.

Reply:
Fixed in https://github.com/vlang/v/pull/5289/commits/ac99373d1ff0b584ccfb7a5ccf27e41b52a85b06